src/content/plugins/split-chunks-plugin.md

```diff
@@ -147,7 +148,7 @@ It can match the absolute module resource path or chunk names. When a chunk name
 
 With the `chunks` option the selected chunks can be configured.
 
-There are 3 values possible `"initial"`, `"async"` and `"all"`. When configured the optimization only selects initial chunks, on-demand chunks or all chunks.
+There are 3 values possible `"initial"`, `"async"` and `"all"`. When configured the optimization only selects initial chunks, on-demand chunks or all chunks. Alternatively, you can provide a function that returns whether to include each chunk (your function is given the chunk object as a parameter).
```

Comment:
It needs modification and rewording. Mention that it accepts a function and string with enum like we do with other params.

The function explanation needs to make more sense, needs example too.

Reply:
The docs currently don't mention that a function is a possible type for this parameter. If they had, it would have saved me quite a lot of time debugging an issue I was having. So any mention is better than what is currently out there.

> Mention that it accepts a function and string with enum like we do with other params

I agree that what I wrote isn't the best this could be, but without references to what you're referring to, I don't exactly know what you're looking for

Reply:
Sorry if that got you confused, here is a reference

For example, this https://webpack.js.org/configuration/optimization/#optimization-runtimechunk

can be set to string object and boolean, all of them have been mentioned and exampled, this needs to be at least of the same format and quality or better.

Function option requires an example, this is a must.

Reply:
otherwise we end up with another issue saying there is no example for the function option :/